app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -82,6 +82,17 @@ class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient:
       .map(_.map(_.toExportsDeclaration))
   }
 
+  def findSavedDeclarations(
```

Comment:
Could potentially reuse the find declarations method (with optional filters & sort params) but this is fine too